### DISCUSSION TITLE
[Esercitazione 15 Novembre 2023] Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#195] [Esercizi - Viaggi/Piazza] [answered]
**Esercizio 4**: sia $G$ un gruppo, $f:G\rightarrow G$ un endomorfismo di $G$ e $H\leq G$
- **(1)** Verificare che, per ogni $a,b\in G$ se $ab=ba$, allora $a^{-1}b=ba{-1}$
- **(2)** Verificare che $K$ è un sottogruppo di $G$
$$K:=\\{x|x\in G,f(xh)=f(hx)\ \forall h\in H\\}$$
> Suggerimento: dovrete utilizzare **(1)**
- **(3)** Dimostrare che se $H$ è normale in $G$ allora $K$ è normale in $G$

> Nota: un omomorfismo di $G$ in $G$ è detto endomorfismo

**Comment by Elia-Belli** [comment]
- **(1)** $a^{-1}b=a^{-1}b\cdot (a \cdot a^{-1})=a^{-1}(ba)a^{-1}=a^{-1}(ab)a^{-1}=ba^{-1}\ \square$
- **(2)** $K\leq G\iff a\cdot b^{-1}\ \forall a,b\in K$
Siano $a,b\in K$, dobbiamo dimostrare che $ab^{-1}\in K\iff f(ab^{-1}h)=f(hab^{-1})\ \forall h\in H$
sappiamo che $f(bh)=f(hb)\Rightarrow f(b)f(h)=f(h)f(b)$ perchè $f$ endomorfismo, quindi $f(h),f(b)\in G$
allora per la **(1)** se $f(b)f(h)=f(h)f(b)\Rightarrow f(b)^{-1}f(h)=f(h)f(b)^{-1}\Rightarrow f(b^{-1}h)=f(hb^{-1})$
Osserviamo che $f(hab^{-1})=f(ha)f(b^{-1})=f(ah)f(b^{-1})=f(a)f(hb^{-1})=f(a)f(b^{-1}h)=f(ab^{-1}h),\forall h\in H$
$\Rightarrow ab^{-1}\in K\ \square$
- **(3)** $K$ è normale in $G\iff aka^{-1}\in K\iff f(aka^{-1}h)=f(haka^{-1})\ \forall h\in H$
siccome $H$ è normale $\Rightarrow ah=ha\Rightarrow^{(1)} a^{-1}h=ha^{-1}$
$f(aka^{-1}h)=f(akha^{-1})=f(ahka^{-1})=f(haka^{-1})\ \square$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/96e7292a-75be-462c-a074-108ff645be02)

**Comment by Elia-Belli** [comment, marked as answer]
## Soluzione Ufficiale
![Screenshot 2023-11-16 103709](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/7791fb76-4f89-4899-bfdb-ce82553524c6)
![Screenshot 2023-11-16 103718](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/6851656d-135c-411f-8a7e-1c89ec9cb4b0)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/babdad15-55d5-4238-aa39-71f300f1e8c8)
